identity/src/main/java/com/stripe/android/identity/navigation/DocSelectionFragment.kt

```diff
@@ -150,8 +150,10 @@ internal class DocSelectionFragment(
             }
         } ?: run {
             // Not possible for backend to send an empty list of allowed types.
-            Log.e(TAG, "Received an empty idDocumentTypeAllowlist.")
-            navigateToDefaultErrorFragment()
+            "Received an empty idDocumentTypeAllowlist.".let { msg ->
+                Log.e(TAG, msg)
```

Comment:
Is the intent to really log this?

Reply:
yes, it technically won't happen as the server is guaranteed not to send us an empty list, but log an error on client side if it happens